Project 2/four-card-feature/style.css

```diff
@@ -0,0 +1,198 @@
+* {
+  box-sizing: border-box;
+  font-size: 15px;
+  font-family: "Poppins", sans-serif;
+  padding-right: 5px;
+  padding-left: 5px;
+}
+
+.container-1 {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: hsl(0, 0%, 98%);
+  margin: 15px 0px;
+}
+
+.par-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+.Item {
+  align-self: center;
+  color: hsl(229, 6%, 66%);
+  padding-right: 10px;
+  text-align: center;
+}
+
+.Item-1 {
+  font-size: 28px;
+  font-weight: 200;
+  margin-bottom: 0px;
+  white-space: nowrap;
+}
+
+.Item-2 {
+  font-size: 28px;
+  font-weight: 600;
+  margin-top: 0px;
+  color: black;
+  margin-bottom: 10px;
+}
+
+.Item-3 {
+  margin-top: 8px;
+  font-size: 17px;
+}
+
+.container-2 {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: center;
+  background-color: hsl(0, 0%, 98%);
+  margin: 15px 0px;
+}
+
+.box {
+  align-self: flex-start;
+  color: hsl(229, 6%, 66%);
+  padding: 0px 0px;
+  border-radius: 10px;
+  margin: 0px 10px 30px 0px;
+  padding: 0px 15px 5px 10px;
+  box-shadow: 1px 1px 10px hsl(229, 6%, 66%);
+}
+
+.Subtitle {
+  font-size: 20px;
+  color: black;
+}
+
+.Items {
+  font-weight: bold;
+}
+
+.Itemz {
+  padding-bottom: 90px;
+}
+
+.box-1 {
+  border: 1px solid hsl(229, 6%, 66%);
+  border-top: 5px solid;
+  border-top-color: hsl(180, 62%, 55%);
+  position: relative;
+}
+
+.box-2 {
+  border: 1px solid hsl(229, 6%, 66%);
+  border-top: 5px solid;
+  border-top-color: hsl(0, 78%, 62%);
+  position: relative;
+}
+
+.box-3 {
+  border: 1px solid hsl(229, 6%, 66%);
+  border-top: 5px solid;
+  border-top-color: hsl(34, 97%, 64%);
+  position: relative;
+}
+
+.box-4 {
+  border: 1px solid hsl(229, 6%, 66%);
+  border-top: 5px solid;
+  border-top-color: hsl(212, 86%, 64%);
+  position: relative;
+}
+
+.picture {
+  position: absolute;
+  bottom: 30px;
+  right: 30px;
+}
+
+@media (min-width: 376px) {
+.box{
+  width: 30vh;
```

Comment:
bardzo ryzykowne, bo uzależniasz szerokość od wysokości czyjegoś ekranu. Jak Ci ktoś wejdzie z turbo długiego telefonu to jest lipa, jak z Nokii 3510i, to też :)